source/upgrade.txt

```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
```

Comment:
```suggestion
If you enable auto updates and don't downgrade |compass|,
```

Reply:
changed to "didn't downgrade" to capture past tense